### DISCUSSION TITLE
Doing 'FetchOrigin' for more than one repository together?

### DISCUSSION THREAD

**Discussion by abhishek-patel-shl** [community/community#45120] [Other Feature Feedback, Questions, & Ideas]
Hello Github Team,

I am using GitHub Desktop to sync GitHub repositories to my local. I have more than 1 GitHub repository for multiple projects (~10).
Currently, I need to select each repository and then do **fetch-origin / pull-request** to make my local sync with all these repositories.

Can I do this efficiently, maybe with one click instead of doing it one-by-one per repository?

Thanks,
Abhishek.

**Comment by jge162** [comment]
@abhishek-patel-shl Unfortunately, GitHub Desktop does not have a built-in feature to allow you to fetch or pull multiple repositories at once. 

otherwise you need to create a script with python or language of choice to run the script to update all of your repositories at once.
